src/main/java/Game.java

```diff
@@ -12,28 +12,34 @@ public boolean hasSomebodyWon() {
         return (hasPlayerWonSet(playerOne, playerTwo) || hasPlayerWonSet(playerTwo, playerOne));
     }
 
-    public boolean hasSomeOneOfThePlayersTwoPointsMore() {
-        return  (Math.abs(this.playerOne.getSetScore() - this.playerTwo.getSetScore()) == 2);
+    public boolean hasSomeoneOfPlayersTwoPointsMore(Player playerWhoIsWinning, Player playerWhoIsLoosing) {
```

Comment:
Typo: `playerWhoIsLoosing` -> `playerWhoIsLosing`

Reply:
Nitpick: maybe rename the method to `hasOneOfThePlayersTwoPointsMore` (it's better to read)

Reply:
It will be dome ;)

Reply:
done*